src/tes/server/task_boltdb.go

```diff
@@ -343,11 +344,10 @@ func (taskBolt *TaskBolt) GetServiceInfo(ctx context.Context, info *ga4gh_task_e
 	//     For example, you can't have multiple S3 endpoints
 	out := map[string]string{}
 	for _, i := range taskBolt.serverConfig.Storage {
-		if i.Local != nil {
+		if reflect.DeepEqual(i.Local, tes.LocalStorage{}) {
```

Comment:
This could do `len(i.Local.AllowedDirs) > 0` and avoid reflect and deep equal and the creation of a throwaway struct.